### ISSUE TITLE
Gateway: Connect to central manager through UI

### ISSUE THREAD

**Issue by DonWillems** [openremote/openremote#161]
When on a gateway installation, show option in the top right header menu that opens the 'Manager interconnect' page.

- [ ] Add item to top right options menu that can be hidden when it is not a gateway installation
- [ ] Create page where the user can add one or more connections to other OR managers.
- [ ] Page shows ingoing and outgoing connections.

**Comment by richturner** [comment]
I think we need an admin page with sub functions of which gateway config would be one....we can't keep adding buttons directly to the header:

- Logs
- Gateways
- Notification Logs (what about send notifications?)
- User management
- Tenant management
- Client management
....

**Comment by DonWillems** [comment]
Sorry only reading this now. I agree that we should combine some/all of those into one page.
Instead of a modal for connecting to another manager I created a page that could be part of a settings page (if navigation were to be added to switch to the other settings pages).

I've called it Interconnect for now. The ability to add multiple managers to connect to if that would make sense later. 
The datatable approach is one similar to what I want to discuss for edit asset and edit agent:
![Manager-19](https://user-images.githubusercontent.com/14177035/78509657-f0ab2c80-778f-11ea-8316-5131103f3206.png)
![Manager-20](https://user-images.githubusercontent.com/14177035/78509660-f1dc5980-778f-11ea-8629-ce5819f90b04.png)
